pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -231,9 +236,74 @@ func (r *reconciler) getOFPriority(rule *CompletedRule) (*uint16, error) {
 	return ofPriority, nil
 }
 
+// BatchReconcile reconciles the desired state of the provided CompletedRules
+// with the actual state of Openflow entries in batch. It should only be invoked
+// if all rules are newly added without last realized status.
+func (r *reconciler) BatchReconcile(rules []*CompletedRule) error {
+	var rulesToInstall []*CompletedRule
+	var priorities []*uint16
+	for _, rule := range rules {
+		if _, exists := r.lastRealizeds.Load(rule.ID); exists {
+			klog.Errorf("rule %s already realized during the initialization phase", rule.ID)
```

Comment:
is this error or warning?

Reply:
As Quan pointed out this should be an error log. BatchReconcile as of now should only be called upon agent restart cases and there should not be lastRealized states